src/open_company_web/components/company_settings.cljs

```diff
@@ -232,13 +232,13 @@
                          :on-click #(.click (sel1 [:input#foce-file-upload-ui--select-trigger]))}
               (dom/i {:class "fa fa-camera"}))
             (dom/button {:class "btn-reset image-url left"
-                         :title "Provide an image link"
+                         :title "Provide a link to your logo"
```

Comment:
Don't forget, when you write up the final notes for this PR, that the logo work is in here too.